subsys/nrf_security/src/drivers/cracen/psa_driver.Kconfig

```diff
@@ -1843,11 +1843,16 @@ config PSA_NEED_CRACEN_KEY_DERIVATION_DRIVER
 		   PSA_NEED_CRACEN_SP800_108_COUNTER_CMAC
 
 config PSA_NEED_CRACEN_KMU_DRIVER
+	bool
+	default y
+	depends on CRACEN_LIB_KMU
```

Comment:
Why loosening of the required Kconfig options here?

Reply:
Because we want to allow using non-encrypted KMU keys and this has less requirements.